_includes/modal.html

```diff
@@ -39,5 +39,7 @@ <h3>Compatibility:</h3>
       {% include partials/compatibility.html%}
       {% include copy-and-download.html %}
     </div>
+    <br>
+    {% include related-discussion.html tag="meshery" %}
```

Comment:
```suggestion
```

You had a better idea than I did, @asubedy. Let's not list related posts inside of the modal (for now anyway), but list them on the single integration page until the modals themselves are better implemented.